imports/izar_protocol_v3.leo

```diff
@@ -0,0 +1,270 @@
+program izar_protocol_v3.aleo {
+    mapping izar_keeper: bool => Keepers20;
+    mapping izar_digest: bool => scalar;
+    // 0: current_epoch, 1: current_keeper_num;
+    mapping izar_config: u8 => u32;
+    mapping nonces: Nonce => bool;
+
+    struct Nonce {
+        chain_id: u32,
+        nonce: u128,
+    }
+
+    inline contract_owner() -> address {
+        return aleo1yu6ssnehj4plgznatenfut3hkrfus26kqd226k99wfx79jqmsuxqfernns;
+    }
+
+    inline unvalid_contract() -> address {
+        return aleo1qqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqq3ljyzc;
```

Comment:
Document how this address comes? `unvalid` should be `invalid` btw.

Reply:
It's the `group::zero()` address. The Leo wallet uses it as a black hole address.